README.md

```diff
@@ -6,13 +6,115 @@
   Plone Content Export and Import
 </h1>
 
-Package supporting the export and import of content, principals, relations, and translations from and to a Plone site.
+Package supporting the export and import of content, principals, relations, translations, discussions, and redirects from and to a Plone site.
 
 ## Introduction
 
 This package is a slimmer version of the awesome [collective.exportimport](https://github.com/collective/collective.exportimport).
 
-While `collective.exportimport` supports older Plone versions and Python 2, and also takes care of data conversion -- i.e.: from Archetypes to Dexterity -- this package focus only on latest Plone and Python.
+While `collective.exportimport` supports older Plone versions and Python 2, and also takes care of data conversion from Archetypes to Dexterity, this package focus only on latest Plone and Python.
+
+
+## Documentation
+
+[`plone.exportimport` documentation](https://6.docs.plone.org/admin-guide/export-import.html)
+
+
+
+## Installation
+
+> [!IMPORTANT]
+> This package supports sites running Plone version 6.0 and above.
+
+> [!IMPORTANT]
+> This package is now included with Plone 6.1 and above by default.
+> The following installation instructions are relevant only for Plone 6.0 sites.
+
+Add `plone.exportimport` to the Plone installation using `pip`.
+
+```shell
+pip install plone.exportimport
+```
+
+
+### Load the package
+
+To make this package available to a Plone installation, you need to load its ZCML configuration.
```

Comment:
The `setup.py` of this package registers a `z3c.autoinclude.plugin` entry point, which means its zcml will get loaded automatically when the Zope instance starts.  So the entire section "Load the package" can be removed.

Reply:
Removed in https://github.com/plone/plone.exportimport/pull/47/commits/06ca6b1f9226e577ce689949fbf4789c0cb34422